R/ebp.R

```diff
@@ -84,15 +84,24 @@
 #' \code{\link[parallelMap]{parallelStart}}.
 #' @param custom_indicator a list of functions containing the indicators to be
 #' calculated additionally. Such functions must and must only depend on the
-#' target variable \code{y} and the \code{threshold}.
+#' target variable \code{y}, \code{weights} and the \code{threshold}.
```

Comment:
It must still be possible without weights, is this the case? And this is not the weights that are used here but the population weights? We should be careful that it is not misunderstood with the sampling weights. Also, maybe we should add ... even if the custom indicator does not use the weights and threshold argument.

Reply:
Ich habe das eingefügt. In der neuen Vignette gibt es dann ein gutes Beispiel dazu. 

\Dropbox\emdi paper\extensions